refinery/core/views.py

```diff
@@ -770,9 +769,10 @@ def get(self, request):
         except Exception:
             filters['group'] = None
 
-        user_data_sets = get_resources_for_user(
-            request.user, 'dataset'
-        ).order_by('-modification_date')
+        user_data_sets = get_objects_for_user(request.user,
+                                              "core.read_meta_dataset",
+                                              accept_global_perms=False)\
+            .order_by('-modification_date')
```

Comment:
I'm not sure if github is doing some odd spacing, but could we clean this up if thats not the case?